src/cli/cli.ts

```diff
@@ -99,7 +119,7 @@ async function main() {
 	for (const route of routes.slice(0, 5)) {
 		const path = route.pairs.map((p, idx) => `${tokenByAddrOrSymbol(route.path[idx]).symbol}:${(p as any).constructor.name}`)
 		console.info(
-			`Output: ${route.outputAmount.shiftedBy(-(outputToken.decimals || 18)).toFixed(6)} -- ` +
+			`Output: ${route.outputAmount.shiftedBy(-(outputToken.decimals || 18)).toFixed(10)} -- ` +
```

Comment:
making optimizations that trade off potential precision, need to see more decimal points to understand whether the loss is acceptable